### DISCUSSION TITLE
copilot.vim stops suggesting after the first use of insert mode.

### DISCUSSION THREAD

**Discussion by pm5k** [community/community#55401] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

I am using v0.9.0 and have installed copilot via Lazy. I did not do any special config, just included it into the require section.
When using the plugin after authentication is successful, I consistently experience this:

- `i` > type stuff, copilot will suggest stuff.
- `n` > do some other stuff after typing.
- `i` > go back to type more stuff, copilot is silent. 0 suggestions for the same actions it suggested on first insert.

I am at a loss, because `LspInfo` just says:

```
 Client: copilot (id: 1, bufnr: [4])
 	filetypes:       
 	autostart:       false
 	root directory:  Running in single file mode.
 	cmd:             node /home/me/.local/share/nvim/lazy/copilot.vim/copilot/dist/agent.js
```

even though it literally just suggested things, which makes me think the `autostart false` is not relevant.

`LspLog` says:

```
.. older stuff here

[START][2023-05-13 18:31:35] LSP logging initiated
[WARN][2023-05-13 18:31:35] ...lsp/handlers.lua:137	"The language server copilot triggers a registerCapability handler despite dynamicRegistration set to false. Report upstream, this warning is harmless"
```

I am sort of stumped now. Because the `Copilot Panel` works even after it stops autosuggesting on insert.
I am running null-ls and treesitter and the only thing I changed from default config for lsp stuff was autoformatting on save using pre-buf writes. So not sure if this tidbit is relevant.

Any suggestions on how to get this to work consistently in insert mode would be greatly appreciated.

Thank you.

**Comment by radimsuckr** [comment]
I experience the same behaviour.

**Comment by IIamaseconds** [comment]
Same issue here

**Comment by f1nwe** [comment]
the same =(

**Comment by sgugler** [comment]
I use Astrovim (and I think it's similar for LuanrVim and LazyVim) I had the same issue. When I do `:Copilot status` it tells me, that there is a conflict with <Tab> as the LSP Lazy Autocomplete thing is mapped to <Tab> and <S-Tab> in `nvim/plugins/cmp.lua`. So I commented out the `["<C-p>"]` and `["<C-n>"]` which are the standard ways of navigating the drop down and rebound the completion from `<Tab>` to `<C-n>` and `<S-Tab>` to `<C-p>`. Now Copilot can use Tab without being killed by cmp. That solved it for me and I can autocomplete copilot with Tab and use the dropdown LSP  suggestions with Ctrl-N, which is intuitive as a vim user.
